DependencyInjection/Configuration.php

```diff
@@ -28,12 +28,18 @@ class Configuration implements ConfigurationInterface
      */
     private $debug;
 
+    /**
+     * @var int
+     */
+    private $symfonyVersion;
+
     /**
      * @param bool $debug
      */
     public function __construct($debug)
     {
         $this->debug = $debug;
+        $this->symfonyVersion = \Symfony\Component\HttpKernel\Kernel::MAJOR_VERSION;
```

Comment:
Total non-sense to store a constant in a class property like this. 

You can do something like this:

```
use Symfony\Component\HttpKernel\Kernel as SymfonyKernel;

// SymfonyKernel::MAJOR_VERSION
```

Reply:
Or you can do this https://github.com/doctrine/DoctrineMigrationsBundle/pull/357/files#diff-850942b3ba24ab03a40aaa81b6152852R184